scripts/zones/The_Boyahda_Tree/mobs/Aquarius.lua

```diff
@@ -11,6 +11,14 @@ entity.onMobInitialize = function(mob)
     mob:setMobMod(xi.mobMod.ADD_EFFECT, 1)
 end
 
+entity.onMobSpawn = function(mob)
+    mob:setMod(xi.mod.SLEEPRES, 100)
+    mob:setMod(xi.mod.DELAY, 1000)
+    mob:addMod(xi.mod.DEFP, 50)
```

Comment:
What are you trying to do here?

Reply:
Increase defense. https://ffxiclopedia.fandom.com/wiki/Aquarius

> Hits fast and hard, but inaccurate. Has high defense.